ods_ci/utils/scripts/testconfig/generateTestConfigFile.py

```diff
@@ -174,8 +174,10 @@ def get_dashboard_url():
     """
     Get dashboard url for the open data science.
     """
-    cmd = "oc get route -A -o json  | jq '.items[].spec.host' | grep 'dashboard'"
-
+    cmd = (
+        "(oc get route -n opendatahub -o json  | jq '.items[].spec.host' | grep odh-dashboard) || "
+        "(oc get route -n redhat-ods-applications -o json  | jq '.items[].spec.host' | grep rhods-dashboard)"
```

Comment:
Please consider also this jq cmd
```
oc get route -A -o json  | jq -r '.items[] | .metadata.namespace as $ns | select($ns == "opendatahub" or $ns == "redhat-ods-applications") | .spec.host'
```

Reply:
i'd prefer fetching less objects than "-A" which fetch all the route from all the ns before filtering with jq

Reply:
ok